components/commons/header/HeaderController.tsx

```
@@ -0,0 +1,64 @@
+    case "LogoHeader":
+      return config.withAlarm ? (
+        <LogoHeader {...commonAlarmProps} />
+      ) : (
+        <LogoHeader />
+      );
+
+    case "BackHeader":
+      const backHeaderProps = {
+        onBackClick: () => router.back(),
+        ...(config.withAlarm && commonAlarmProps),
+      };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)